skyportal/app_server.py

```diff
@@ -472,6 +474,10 @@ def log_request(self, handler):
     (r'/api/internal/notifications/all', BulkNotificationHandler),
     (r'/api/internal/ps1_thumbnail', PS1ThumbnailHandler),
     (r'/api/internal/recent_gcn_events', RecentGcnEventsHandler),
+    (
+        r'/api/internal/sources(/[0-9A-Za-z-_\.\+]+)/image_analysis',
+        ImageAnalysisHandler,
+    ),
```

Comment:
This means it is not accessible to the users?

Reply:
Yes. I believe this was the result of a discussion between @mcoughlin and Josh

Reply:
Yeah only through the app.